environment/cpu/src/cloud/benchflow/monitors/cpu.go

```diff
@@ -82,6 +82,7 @@ func monitorStats(container Container) {
 				return
 			default:
 				stat := (<-container.statsChannel)
+				if(stat == nil) {panic("Received Nil from container")}
```

Comment:
@VincenzoFerme Because a nil means the socket isn't receiving anything, which seems to imply something has gone seriously wrong with the docker socket.